bin.js

```diff
@@ -16,7 +16,9 @@ var opts = {
   force:     args.f || args.force,
   sockets:   args.s || args.sockets,
   verbose:   args.verbose === undefined ? process.stdin.isTTY : args.verbose,
-  frequency: args.frequency ? +args.frequency : null
+  frequency: args.frequency ? +args.frequency : null,
+  proxy:     args.proxy ? args.proxy : null,
+  strictSSL: args['strict-ssl'] !== null ? args['strict-ssl'] !== 'false' : null
```

Comment:
if you just pass `--no-strict-ssl` then `args['strict-ssl']` will be set to false so we could just set the `strictSSL` option directly to `args['strict-ssl']`